examples/react/README.md

```diff
@@ -0,0 +1,11 @@
+# React | Typescript | TailwindCSS
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
```

Comment:
I think it would be handy to have explicit instructions that the node app needs to be run as well for it to work properly